app/controllers/api/v1/orders_controller.rb

```diff
@@ -118,6 +118,7 @@ def order_response(records)
           include_packages: true,
           include_order: false,
           include_images: true,
+          include_messages: params["include_messages"],
```

Comment:
```suggestion
          include_messages: bool_param(:include_messages, false),
```

The query params are all strings, so if `include_message` is a string which says "false" it will still evaluate to something truthy. (same thing below)